### PR TITLE
[WIP] Add json support

### PR DESCRIPTION
**Pull request by ragdubizzle**

### CODE REVIEW

File: eav/models.py [hunk 1]
```diff
@@ -452,6 +457,18 @@ class Meta:
...
+    @property
+    def value_json(self):
+        if self.value_text:
+            return json.loads(self.value_text)
+        else:
+            return {}
+
+    @value_json.setter
+    def value_json(self, new_value):
+        self.value_text = json.dumps(new_value)
```
Comment (mtahirtariq) [inline eav/models.py:470]: May be we should check if it is already string then do not convert it?

File: eav/models.py [hunk 2]
```diff
@@ -266,6 +270,7 @@ def get_validators(self):
             'object':      validate_object,
             'enum':        validate_enum,
             'enum_multi':  validate_enum_multi,
+            'json':        value_json,
```
Comment (mtahirtariq) [inline eav/models.py]: This is validators dict, the name should be validate_json?

Comment (ragdubizzle) [reply eav/models.py]: Yeah it is a typo.

File: eav/models.py (hunk 1 above)
Comment (ragdubizzle) [reply eav/models.py:470]: It should be always dict no

Comment (mtahirtariq) [reply eav/models.py:470]: We can force it to be dict, both are fine with me.

**Review by mtahirtariq — APPROVED** [review]

**Pull request merged**